qa/tasks/s3tests.py

```diff
@@ -582,6 +582,16 @@ def task(ctx, config):
               cloudtier_tests: True
               rgw_server: client.0
 
+    To run any iam-tests don't forget to set a config variable named 'iam_tests' to 'True' as follows::
+
+        tasks:
+        - ceph:
+        - rgw: [client.0]
+        - s3tests:
+            client.0:
+              iam_tests: True
+              rgw_server: client.0
```

Comment:
you need to have `extra_args: ["test_of_iam"]` in order to run the iam tests specifically. So, please include this line here.

Reply:
What's the difference between `extra_args` and `extra_attrs` for the config here?

Reply:
@alimaredia It was a typo by me, here it's `extra_attrs` only.